docs/getting-started/docker.mdx

```diff
@@ -33,9 +33,14 @@ docker run -d \
   --name jellyseerr \
   -e LOG_LEVEL=debug \
   -e TZ=Asia/Tashkent \
-  -e PORT=5055 `#optional` \
+  -e PORT=5055 \
   -p 5055:5055 \
   -v /path/to/appdata/config:/app/config \
+  --health-cmd "wget --no-verbose --tries=1 --spider http://localhost:5055 || exit 1" \
+  --health-start-period 20s \
+  --health-timeout 3s \
+  --health-interval 15s \
+  --health-retries 3 \
```

Comment:
Shouldn't healthcheck be optional as well. It might confuse complete beginners when its too complicated

Reply:
In my view, the docker run command is intended for systemd users who prefer not to use Docker Compose or who choose to use systemd to start their containers. Otherwise, I would suggest moving it to another section, such as 'How to try out Jellyseerr'.